lib/ZumoOta/src/FileManager.h

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+    DESCRIPTION
+*******************************************************************************/
+/**
+ * @brief  ZumoOta application
+ * @author Decareme Pauline Ngangnou <ngandeca@yahoo.fr>
+ * @{
+ */
+#ifndef FILEMANAGER_H
+#define FILEMANAGER_H
+
+/******************************************************************************
+ * Includes
+ *****************************************************************************/
+#include <Arduino.h>

```

Comment:
why do you include this here ? Remove and add to cpp if needed there.

Reply:
I will perform it with the next commit.